pkg/platform/runtime/buildscript/buildscript.go

```diff
@@ -1,102 +1,119 @@
 package buildscript
 
 import (
-	"errors"
-	"os"
-	"path/filepath"
-
-	"github.com/ActiveState/cli/internal/constants"
-	"github.com/ActiveState/cli/internal/errs"
-	"github.com/ActiveState/cli/internal/fileutils"
-	"github.com/ActiveState/cli/internal/logging"
-	"github.com/ActiveState/cli/pkg/platform/api/graphql/model/buildplanner"
-	"gopkg.in/yaml.v2"
+	"bytes"
+	"io"
 )
 
-type DoesNotExistError struct{ error }
+type Script struct {
+	Let *Let `parser:"'let' ':' @@"`
```

Comment:
These struct tags utilize https://github.com/alecthomas/participle, a Go package for dead-simple parsing into structures.